src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -759,14 +798,16 @@ public C(string s)
 
         S = s;
     }
-}", parameters: new TestParameters(options:
-    Option(CSharpCodeStyleOptions.PreferThrowExpression, CodeStyleOptions2.FalseWithSilentEnforcement)));
+}"
+            };
+            test.Options.Add(CSharpCodeStyleOptions.PreferThrowExpression, CodeStyleOptions2.FalseWithSilentEnforcement);
+            await test.RunAsync();
```

Comment:
`Options` can be added in the object initializer.

💡 This suggestion applies to multiple tests.

```suggestion
}",
                Options = { (CSharpCodeStyleOptions.PreferThrowExpression, false) },
            }.RunAsync();
```

Reply:
TIL! Thank you. Is that new?

Completion certainly doesn't help discoverability of this one, it only suggests `new XXX` which is a compile error.